script/testing/oltpbench/test_case_oltp.py

```diff
@@ -117,7 +119,7 @@ def run_post_test(self):
             # publish results
             if self.publish_results:
                 report(self.publish_results, os.path.join(
-                    os.getcwd(), "oltp_result",self.filename_suffix))
+                    os.getcwd(), "oltp_result",self.filename_suffix),self.publish_username,self.publish_password,self.query_mode)
```

Comment:
Maybe you can use pep8 and flake8 to check the format. I think they can automatically fix the formatting issues.

Reply:
Sorry, I just realized we don't have pep8 in this repo right?